theano/sandbox/multinomial.py

```diff
@@ -79,12 +79,12 @@ def c_code(self, node, name, ins, outs, sub):
         return """
         if (PyArray_NDIM(%(pvals)s) != 2)
         {
-            PyErr_Format(PyExc_TypeError, "pvals wrong rank");
+            PyErr_Format(PyExc_TypeError, "pvals ndim should be 2");
```

Comment:
Oh my! :sweat: Sorry I didn't realized the dark green region was not the entire string. My bad!